src/shared/install.c

```diff
@@ -1050,25 +1050,32 @@ static int config_parse_also(
         assert(rvalue);
 
         for (;;) {
-                _cleanup_free_ char *word = NULL;
+                _cleanup_free_ char *word = NULL, *printed = NULL;
 
                 r = extract_first_word(&rvalue, &word, NULL, 0);
                 if (r < 0)
                         return r;
                 if (r == 0)
                         break;
 
-                r = install_info_add(c, word, NULL, &alsoinfo);
+                r = install_full_printf(info, word, &printed);
+                if (r < 0)
+                        return r;
+
+                if (!unit_instance_is_valid(printed))
+                        return -EINVAL;
```

Comment:
Hm, maybe. It's possible that we would try to create a symlink with the template name... I'll check what happens.